New package: ShaneDBSCAN v1.0.0

JuliaRegistrator opened the pull request:
- Registering package: ShaneDBSCAN
- Repository: https://github.com/kchu25/ShaneDBSCAN.jl
- Created by: @kchu25
- Version: v1.0.0
- Commit: 667fdd0ddb9cb56e345178d6edb9fc8a64c4ef81
- Reviewed by: @kchu25
- Reference: https://github.com/kchu25/ShaneDBSCAN.jl/commit/667fdd0ddb9cb56e345178d6edb9fc8a64c4ef81#commitcomment-147398480
- Description: non-parametric clustering the quick way 
<!-- bf0c69308befbd3ccf2cc956ac8a46712550b79fc9bfb5e4edf8f833f05f4c18b06eddad8845b45beb9f45c2b8020dd6445db102156fc50d743b00e38847ef5e88e316c8076c069b51bf4acabb8a69653429c3a3eadbbe2f5fd4c1672d5202bc1638b4ddb6b768cc1e60304455b41ca7acc47d6f8d1eebe92867722ef2adeb6da819c7beccbe5c0d854bf2f462b7090135c9a3942471d39287b4dc17d2fca545bdbe3858a23634b613dc76dfd420deb137264f83bd6337d8077a0d3913c5c47a0b369c0e6f55355e7024c34a58105423b6b44f9f5bbf1a96118ed9ed1e12ccdc -->

goerz commented:
Thank you for submitting your package! However, please make sure to add some documentation before registering. [At the very least](https://discourse.julialang.org/t/when-does-it-make-sense-to-post-a-new-package-to-the-julia-repository/108701/4), that would be a description of the package's purpose and a small usage example in the README

JuliaRegistrator committed:
New package: ShaneDBSCAN v1.0.0

UUID: cf00368d-0432-4e68-9dac-168cccbdfede
Repo: https://github.com/kchu25/ShaneDBSCAN.jl.git
Tree: 0094c0e8fdc010227915ef552e4a7a4f6d57f22f

Registrator tree SHA: 17aec322677d9b81cdd6b9b9236b09a3f1374c6a
